python/ray/tune/utils/__init__.py

```diff
@@ -1,14 +1,14 @@
 from ray.tune.utils.util import (
```

Comment:
When do/don't we add references in this file?  For example, the `atomic_save` function is not defined here.

Reply:
I don't think there's a convention for this either, I think `atomic_save` was probably just forgotten to be included. @richardliaw?